pkg/daemon/create.go

```diff
@@ -82,13 +93,31 @@ func Create(env envvar.Env) (*server.Server, *worker.Worker, error) {
 
 	// --------------------------------------------------------------------- //
 
+	var dae *Daemon
+	{
+		dae = &Daemon{
+			env: env,
+			lis: lis,
+			loc: loc,
+			log: log,
+			rec: rec,
+			red: red,
+			res: res,
+			sto: sto,
+		}
+	}
+
+	return dae
```

Comment:
Having this encapsulated in a struct allows us to call dedicated methods in a declarative way. I tried to patch the issue first with some boolean flags but I didn't like it very much. So here we go.